web/packages/teleport/src/Discover/SelectResource/databases.tsx

```diff
@@ -365,8 +365,8 @@ export const DATABASES: ResourceSpec[] = [
       location: DatabaseLocation.Aws,
       engine: DatabaseEngine.AuroraMysql,
     },
-    name: 'Aurora MySQL/MariaDB',
-    keywords: [...awsKeywords, 'aurora mysql mariadb'],
+    name: 'RDS Aurora MySQL/MariaDB',
```

Comment:
I don't think there is such a thing for Aurora MariaDB. I wonder if we want to match what RDS presents like `Aurora (MySQL compatible)`
<img width="790" alt="Screenshot 2024-10-31 at 4 57 45 PM" src="https://github.com/user-attachments/assets/fb2960db-ccfb-4a7e-9c5f-88b2a03acebe">